src/components/FilterPanel/CheckboxGroup.jsx

```diff
@@ -0,0 +1,31 @@
+import { Checkbox, FormControlLabel, FormGroup } from '@mui/material';
+
+export const CheckboxGroup = ({
+  section,
+  getStateBySection,
+  getHandlerBySection,
+}) => {
+  const state = getStateBySection(section.id);
+  const handler = getHandlerBySection(section.id);
```

Comment:
You could just pass the state and the handler instead of passing the `getStateBySection` and the `getHandlerBySection` functions